samples/apache-others/healthcheck/exploit/doit.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pwn import *
```

Comment:
this should use pwnlib, then you can remove the NOTERM hacks

Reply:
I'll push this after #40 although we probably should migrate the other ones too then

Reply:
I need to deploy this to test something else, so I'll push now and we can adjust to pwnlib after #40 is submitted then :)